rides/pings.py

```diff
@@ -0,0 +1,46 @@
+import requests
+from rides import app
+
+def send_join(to, join, ride):
+    pings_enabled = app.config.get["PINGS_ENABLED"]
+    pings_join_route = app.config.get["PINGS_JOIN_ROUTE_UUID"]
+    pings_token = app.config.get["PINGS_TOKEN"]
+    if pings_join_route is None or pings_join_route == "" or pings_token is None or pings_token == "" or not pings_enabled:
+        print("Pings is not configured")
+        return
```

Comment:
I would check if Pings is not enabled first, so you don't spam the log if it's disabled:
```
    if not app.config.get["PINGS_ENABLED"]:
        return
```
Secondly, you can simplify this config check somewhat as both `None` and empty strings are falsey:
```
    pings_join_route = app.config.get["PINGS_JOIN_ROUTE_UUID"]
    pings_token = app.config.get["PINGS_TOKEN"]

    if not pings_join_route or not pings_token:
        print("Pings is enabled but not configured, missing PINGS_TOKEN and PINGS_JOIN_ROUTE_UUID")
        return
```